app/src/main/java/ch/epfl/skysync/viewmodel/TimerViewModel.kt

```diff
@@ -25,6 +25,8 @@ class TimerViewModel : ViewModel() {
   private var lastTimestamp = 0L
   private val _counter = MutableStateFlow(0L)
 
+  val rawCounter = _counter.stateIn(viewModelScope, started = WhileUiSubscribed, initialValue = 0)
```

Comment:
Actually I don't know the difference between the two, but I can change it

Reply:
with stateIn we need an active subscriber i.e. a composable that is reading data so that the state is updated whereas a normal flow is updated in any case.